python/rikai/internal/reflection.py

```diff
@@ -22,3 +22,30 @@ def find_class(class_name: str):
     module, cls = class_name.rsplit(".", 1)
     mod = importlib.import_module(module)
     return getattr(mod, cls)
+
+
+def check_class(class_name: str):
+    module, cls = class_name.rsplit(".", 1)
```

Comment:
if input doesn't contain "." this is going to raise a cryptic "not enough values to unpack". You may want to check explicitly and raise an error with a clearer message. 
The same holds for the other two functions as well.